src/lib/ControlButton.svelte

```diff
@@ -9,7 +9,7 @@
   export { classNames as class };
 </script>
 
-<button type="button" {title} on:click>
+<button type="button" {title} on:click {...$$props}>
```

Comment:
Let's use `...$$restProps` here instead. Also did it not work to just pass `rounded-full` in the `class` property?